apps/web/src/views/PancakeSquad/hooks/useUserInfos.tsx

```diff
@@ -25,7 +25,7 @@ const useUserInfos = ({ account, refreshCounter, setCallback }) => {
               } as const),
           )
 
-          const client = publicClient({ chainId: ChainId.BSC })
+          const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```